task/orka-full/0.1/README.md

```diff
@@ -0,0 +1,177 @@
+
+To uninstall from the `default` namespace, run the script with the `-d` or `--delete` flag:
+
+```sh
+./install.sh --delete
+```
+
+**Custom namespace installation**
+
+To install in a custom namespace, run the following command against your preferred namespace and your actual Orka API endpoint:
+
+```sh
+NAMESPACE=tekton-orka ORKA_API=http://10.221.188.100 ./install.sh --apply
+```
+
+To uninstall from a selected namespace, run the script with the `-d` or `--delete` flag against the namespace:
+
+```sh
+NAMESPACE=tekton-orka ./install.sh --delete
+```
+
+### Storing your credentials
+
+The provided `Tasks` look for two Kubernetes secrets that store your credentials: `orka-creds` for the Orka user and `orka-ssh-creds` for the SSH credentials. In the current setup, both secrets have `username` and `password` keys.
+
+These defaults exist for convenience and you can change them using the available [`Task` parameters](#Configuring-Secrets-and-Config-Maps).
+
+**Script setup**
+
+You need to create Kubernetes secrets to store the Orka user credentials and the base image's SSH credentials.
+
+To create a Kubernetes secret in the `default` namespace of your cluster, run the following commands:
+
+```sh
+EMAIL=<email> PASSWORD=<password> ./add-orka-creds.sh --apply
+SSH_USERNAME=<username> SSH_PASSWORD=<password> ./add-ssh-creds.sh --apply
+```
+
+To remove the secrets from the `default` namespace, run:
+
+```sh
+./add-orka-creds.sh --delete
+./add-ssh-creds.sh --delete
+```
+
+To create a Kubernetes secret in a custom namespace, run the following commands against your preferred namespace:
+
+```sh
+NAMESPACE=tekton-orka EMAIL=<email> PASSWORD=<password> ./add-orka-creds.sh --apply
+NAMESPACE=tekton-orka SSH_USERNAME=<username> SSH_PASSWORD=<password> ./add-ssh-creds.sh --apply
+```
+
+To remove the secrets from the custom specify, run the following commands against the namespace:
+
+```sh
+NAMESPACE=tekton-orka ./add-orka-creds.sh --delete
+NAMESPACE=tekton-orka ./add-ssh-creds.sh --delete
+```
+
+**Manual setup**
+
+If you want to create the Kubernetes secrets manually, you can use the following example configuration. Make sure to provide the correct credentials for your Orka environment and the base image.
+
+
+```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orka-creds
+type: Opaque
+stringData:
+  email: tekton-svc@macstadium.com
+  password: p@ssw0rd
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orka-ssh-creds
+type: Opaque
+stringData:
+  username: admin
+  password: admin
+```
+
+**Using an SSH key**
+
+If using an SSH key to connect to the VM instead of an SSH username and password, complete the following:
+
+1. Copy the public key to the VM and commit the base image.
+2. Store the username and private key in a Kubernetes secret:
+
+```sh
+kubectl create secret generic orka-ssh-key --from-file=id_rsa=/path/to/id_rsa --from-literal=username=<username>
+```
+
+See also: [`use-ssh-key`](samples/use-ssh-key.yml) example
```

Comment:
```suggestion
See also: [`use-ssh-key`](samples/use-ssh-key.yaml) example
```

Reply:
Had to rename the files from `.yml` to `.yaml` to conform to Tekton's standards ;)